bw2calc/multi_lca.py

```diff
@@ -348,9 +348,8 @@ def lci_calculation(self) -> None:
 
         """
         count = len(self.dicts.activity)
-        solutions = spsolve(
-            self.technosphere_matrix, np.vstack([arr for arr in self.demand_arrays.values()]).T
-        )
+        demand_matrix = np.vstack([arr for arr in self.demand_arrays.values()]).T
```

Comment:
Not sure we need this to be a separate variable?

Reply:
Separating this is certainly not necessary, I just thought it'd be more readable. In fact, the original commit [801f621](https://github.com/brightway-lca/brightway2-calc/pull/104/commits/801f6218f3925a29df81f711a4ec5bbfd6a4f170) in this PR didn't have this. I can roll this back of you prefer not separating it.